### DISCUSSION TITLE
Konfiguration HistoryGraph

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#1234] [Q&A]
Ich tu mich noch ein bisschen schwer mit der Konfiguration vom HistoryGraph.
Ich möchte gerne, dass mir x Wochen im Rückblick angezeigt werden.
Dabei sollen alle Daten schon geladen werden ohne dass ich den Daten Zoom Slider verschieben muss.

**Comment by mcuiobroker** [comment]
![image](https://user-images.githubusercontent.com/84818934/144280274-fff48ba3-9cf1-4782-98af-7dd046964783.png)
Kannst ja hier mal schauen, wenn was fehlt direkt ansprechen
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/historygraph/modul-parameter

**Reply by AFPass** [reply]
Hmm. Merkwürdig. Hatte ich nicht. Neu angelegt, jetzt ist es da. 

Danke.

**Comment by AFPass** [comment]
Noch eine Frage.
Ich habe zwei Daten die dargestellt werden sollen. Das klappt auch. Nun möchte ich den Bereich von Y begrenzen.
Dazu habe ich bei "Konfiguration y-Achse folgendes eingetragen:
[{"min":"dataMin-5","max":"dataMax+5"},{"min":"dataMin-5","max":"dataMax+5"}]

Leider werden mir nun keine Bereiche bei y mehr angezeigt.

Ein DP im Chart mit {"min":"dataMin-5","max":"dataMax+5"} funktioniert.

**Comment by mcuiobroker** [comment]
Hast du mal versucht nur einen Eintrag im Array zu machen:
```
[{"min":"dataMin-5","max":"dataMax+5"}]
```

**Reply by AFPass** [reply]
Sieht noch genauso aus. Ohne Werte an Y.

**Reply by mcuiobroker** [reply]
Dann musst du die +/-5 rausnehmen?
```[{"min":"dataMin","max":"dataMax"}]```

**Reply by AFPass** [reply]
Dann sehe ich nur links die Skala. Ausgehend vom letzten Wert.

Auch bei [{"min":"dataMin","max":"dataMax"},{"min":"dataMin","max":"dataMax"}] sehe ich nur links eine Skala.
